src/settingsMap.ts

```diff
@@ -0,0 +1,38 @@
+class SettingsMap {
+    private settings: Map<string, Map<string, string>>
+
+    constructor() {
+        this.settings = new Map<string, Map<string, string>>()
+    }
+
+    public get(guildId: string, setting: string) : string {
+		return this.settings.get(guildId)?.get(setting) ?? ""
+	}
+
+	public initGuildSettings(guildId : string){
```

Comment:
We could create a default setting object that we can assign to each guild by default, with the types being `number | string | boolean`. This can be done later though.